lib/takeaway.rb

```diff
@@ -0,0 +1,34 @@
+require_relative "../lib/basket.rb"
+
+class Takeaway
+
+  attr_reader :menu
+
+  DEFAULT_MENU = {
+    "Burger" => { name: "Burger", price: 3 },       
+    "Chips" => { name: "Chips", price: 1 },
+    "Kebab" => { name: "Kebab", price: 5 } }
+
+  def initialize(menu: DEFAULT_MENU)
+    @menu = menu
+    @basket = Basket.new
+  end
+  
+  def menu
+    string = ""
+    @menu.keys.each do |item|
+      string += "#{@menu[item][:name]} : £#{@menu[item][:price]}\n"
+    end
+    return string
+  end
+
+  def addBasket(food)
```

Comment:
OK, and your hash of hashes allows you to select the correct menu item in this nice way